File: Swift/Sort/Sort/Number.swift
import Cocoa

// Number
// Created by Paul Kraft on 04.12.15.

class Number : CompareElement
{
	private let POWER = 9
	init()
	{
		super.init(Int(arc4random_uniform(UInt32(pow(Double(10),Double(POWER))))))
	}
}

File: Java/LinkedList/Number.java
package SortingAlgs;
import java.util.Random;
import java.math.*;

// Number
// Created by Paul Kraft on 30.11.15.

public class Number {
	private int number;
	private static int POWER = 10;
	public Number()
	{
		this.number = (new Random()).nextInt((int)Math.pow(10.0,(double)(POWER)));
	}
	
	public String toString()
	{
		String a = number + "";
		while(a.length() < POWER)
		{
			 a = "0" + a;
		}
		return a;
	}
}